hal/x86_fsp_tgl_loader.c

```diff
@@ -0,0 +1,95 @@
+/* x86_fsp_qemu_loader.c
```

Comment:
This is needed because we need alternative (empty) hal functions in stage1. Let me know if you suggest a different way to have hal_ functions for stage1

Reply:
the hal used for wolfboot kontron target (hal/kontorn_vx3060_s2.c) is already almost empty. Include that object as hal/$(TARGET).c  and guard the single function that isn't empty (it may be already guarded by the __WOLFBOOT).

Reply:
Agreed offline to keep the separate _loader hal, but renaming to the correct target `hal/kontron_vx3060_s2_loader.c`